index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.